packages/@aws-cdk/aws-stepfunctions-tasks/test/glue/integ.start-job-run.ts

```diff
@@ -0,0 +1,63 @@
+import * as glue from '@aws-cdk/aws-glue';
+import * as iam from '@aws-cdk/aws-iam';
+import * as assets from '@aws-cdk/aws-s3-assets';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as path from 'path';
+import { GlueStartJobRun } from '../../lib/glue/start-job-run';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn>
+ * * aws stepfunctions describe-execution --execution-arn <execution arn created above>
+ * The "describe-execution" call should eventually return status "SUCCEEDED".
+ * NOTE: It will take up to 15 minutes for the step function to complete due to the cold start time
+ * for AWS Glue, which as of 02/2020, is around 10-15 minutes.
+ */
```

Comment:
Assume you've run and verified this. 

Feel free to drop the 'do-not-merge' if/when you have.

Reply:
confirming that I ran this, invoked and completed it successfully. 
Also went into the console and checked it out.